project/AutomaticModuleName.scala

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <http://www.lightbend.com>
```

Comment:
```suggestion
 * Copyright (C) 2009-2019 Lightbend Inc. <http://www.lightbend.com>
```

Will this file also be updated by the hearder plugin? I'm wondering how it will behave with new files that do not have